e2e_deployment_files/long_haul_deployment.template.json

```diff
@@ -26,10 +26,10 @@
             },
             "env": {
               "experimentalfeatures__enabled": {
-                  "value": "true"
+                "value": "true"
               },
               "experimentalfeatures__enableMetrics": {
-                  "value": "true"
+                "value": "true"
```

Comment:
this is removed recently, please confirm with Lee and help to clean this up (only long haul deployment json, no need to fix connectivity deployment json since this PR doesn't touch them).